### DISCUSSION TITLE
how to unformat/unmask?

### DISCUSSION THREAD

**Discussion by laptopmutia** [scurker/currency.js#426] [Q&A]
sometimes I want to get the integer value of a currency object

**Comment by scurker** [comment]
There's a couple of different ways you can do this:

If the value is still a currency value, `.value` will return the decimal value of the currency and `.intValue` will return the integer value of the currency:

```js
const c1 = currency(1234.56); 
console.log(c1.value); // => 1234.56
console.log(c1.intValue); // => 123456
```

If the value has already been formatted, you can pass the formatted value into a currency object to get the numeric values:

```js
const c1 = currency('$1,234.56'); 
console.log(c1.value); // => 1234.56
console.log(c1.intValue); // => 123456
```
